source/features/highest-rated-comment.tsx

```diff
@@ -76,6 +76,8 @@ function linkBestComment(bestComment: HTMLElement): void {
 
 		// Copy avatar but link it to the comment
 		const avatar = select('.TimelineItem-avatar', bestComment)!.cloneNode(true);
+		// Remove the check icon from the preview #3338
+		select('.octicon-check.text-green', avatar)!.remove();
```

Comment:
This is in the middle of the “ Copy avatar but link it to the comment” operation. Move it 4 lines down, with an extra line